### ISSUE TITLE
Trinity tool definition installation issue

### ISSUE THREAD

**Issue by djevo1** [galaxyproject/tools-iuc#989]
There appears to be an issue with outdated links in some trinity dependencies below are the issues.

Edger package

```
 File "/home/eclark28/galaxy_production/galaxy/lib/tool_shed/galaxy_install/install_manager.py", line 125, in install_and_build_package_via_fabric
    tool_dependency = self.install_and_build_package( install_environment, tool_dependency, actions_dict )
  File "/home/eclark28/galaxy_production/galaxy/lib/tool_shed/galaxy_install/install_manager.py", line 83, in install_and_build_package
    initial_download=True )
  File "/home/eclark28/galaxy_production/galaxy/lib/tool_shed/galaxy_install/tool_dependencies/recipe/recipe_manager.py", line 31, in execute_step
    initial_download=initial_download )
  File "/home/eclark28/galaxy_production/galaxy/lib/tool_shed/galaxy_install/tool_dependencies/recipe/step_handler.py", line 1362, in execute_step
    self.url_download( work_dir, filename, url, extract=False, checksums=checksums )
  File "/home/eclark28/galaxy_production/galaxy/lib/tool_shed/galaxy_install/tool_dependencies/recipe/step_handler.py", line 184, in url_download
    raise Exception( err_msg )

Error downloading from URL
https://depot.galaxyproject.org/software/limma/limma_3.28.2_src_all.tar.gz:
HTTP Error 404: Not Found

```

Limma

```
  File "/home/eclark28/galaxy_production/galaxy/lib/tool_shed/galaxy_install/install_manager.py", line 125, in install_and_build_package_via_fabric
    tool_dependency = self.install_and_build_package( install_environment, tool_dependency, actions_dict )
  File "/home/eclark28/galaxy_production/galaxy/lib/tool_shed/galaxy_install/install_manager.py", line 83, in install_and_build_package
    initial_download=True )
  File "/home/eclark28/galaxy_production/galaxy/lib/tool_shed/galaxy_install/tool_dependencies/recipe/recipe_manager.py", line 31, in execute_step
    initial_download=initial_download )
  File "/home/eclark28/galaxy_production/galaxy/lib/tool_shed/galaxy_install/tool_dependencies/recipe/step_handler.py", line 1362, in execute_step
    self.url_download( work_dir, filename, url, extract=False, checksums=checksums )
  File "/home/eclark28/galaxy_production/galaxy/lib/tool_shed/galaxy_install/tool_dependencies/recipe/step_handler.py", line 184, in url_download
    raise Exception( err_msg )

Error downloading from URL
https://depot.galaxyproject.org/software/limma/limma_3.28.10_src_all.tar.gz:
HTTP Error 404: Not Found
```

Trinity

```
File "/home/eclark28/galaxy_production/galaxy/lib/tool_shed/galaxy_install/install_manager.py", line 125, in install_and_build_package_via_fabric
    tool_dependency = self.install_and_build_package( install_environment, tool_dependency, actions_dict )
  File "/home/eclark28/galaxy_production/galaxy/lib/tool_shed/galaxy_install/install_manager.py", line 110, in install_and_build_package
    initial_download=False )
  File "/home/eclark28/galaxy_production/galaxy/lib/tool_shed/galaxy_install/tool_dependencies/recipe/recipe_manager.py", line 31, in execute_step
    initial_download=initial_download )
  File "/home/eclark28/galaxy_production/galaxy/lib/tool_shed/galaxy_install/tool_dependencies/recipe/step_handler.py", line 1362, in execute_step
    self.url_download( work_dir, filename, url, extract=False, checksums=checksums )
  File "/home/eclark28/galaxy_production/galaxy/lib/tool_shed/galaxy_install/tool_dependencies/recipe/step_handler.py", line 201, in url_download
    raise Exception( 'Given sha256 checksum does not match with the one from the downloaded file (%s != %s).' % (downloaded_checksum, expected) )

Given sha256 checksum does not match with the one from the downloaded file (1191c0022bcba15e5b896f6197ac91bc7fb4679e150e9c32ad4181a812ec74f1 != f16b6316fdbd7ab0077644d41a3286a688b0b75cc34686383e10f139f0955ae1)

```

**Comment by abretaud** [comment]
Hi,
The last error is the same as #959.
For the limma problems, the package disappeared from depot during The Big Disaster.
I think we need to switch to more recent releases of limma and BioBase packages, I can take a look at this, but not right now sorry

**Comment by djevo1** [comment]
Ok thank you.

**Comment by abretaud** [comment]
It should be fixed now with galaxyproject/cargo-port/pull/107 galaxyproject/cargo-port/pull/105 #1028 #1029 #1031 
(merci @lecorguille !)
